app/main/posts/detail/detail.html

```diff
@@ -116,15 +117,25 @@ <h3 class="listing-heading" translate="app.tasks">Tasks</h3>
           </div>
           <div
             ng-repeat="(key,value) in post.values"
-            ng-if="form_attributes[key].type !== 'media' && !isPostValue(key) && form_attributes[key] && showType(form_attributes[key].type)"
             ng-show="form_attributes[key].form_stage_id === visibleTask"
             >
             <post-value
+              ng-if="form_attributes[key].type !== 'media' && !isPostValue(key) && form_attributes[key] && showType(form_attributes[key].type)&& form_attributes[key].input !== 'tags'"
```

Comment:
Can we move this if condition to a function on the controller?
In actually looks like `showType` might already be doing that and the other conditions should get merged in?